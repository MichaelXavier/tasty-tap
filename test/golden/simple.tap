TAP version 13
1..2
ok 1 - some example tests.passes
not ok 2 - some example tests.fails
# test/Main.hs:56:
# this was doomed
