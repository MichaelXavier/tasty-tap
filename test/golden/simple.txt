TAP version 13
1..2
ok 1 - some example tests/passes
not ok 2 - some example tests/fails
# this was doomed
